<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>QRDialog</class>
  <widget class="QDialog" name="QRDialog">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>500</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>QR Code Generator</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
      <item>
        <widget class="QComboBox" name="boxComboBox"/>
      </item>
      <item>
        <widget class="QLabel" name="qrCodeLabel">
          <property name="alignment">
            <set>Qt::AlignCenter</set>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QPushButton" name="downloadButton">
          <property name="text">
            <string>Download QR Code</string>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <resources/>
  <connections/>
</ui>